project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.5.5
\ No newline at end of file
+sbt.version=1.7.1
```

Comment:
afaik latest is `1.8.0` but I guess `1.7.1` is the latest version compatible with this scala version right?

Reply:
I will check that, indeed there is a newer version